3.evaluate_model/scripts/nbconverted/confusion_matrices.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ### Import Libraries
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+from joblib import load
+
+import sys
+sys.path.append("../utils")
+from split_utils import get_features_data
+from train_utils import get_dataset
+from evaluate_utils import model_cm
+
+
+# ### Load Datasets
+
+# In[2]:
+
+
+# load features data from indexes and features dataframe
+data_split_path = pathlib.Path("../1.split_data/indexes/data_split_indexes.tsv")
```

Comment:
While not added to this review, I noticed that `data_split_indexes.tsv` has an unlabeled column which might be interpreted as an index (in addition to the column called "index"). Would this be cause for any issues here? I bring this up just in case there's a chance of discrepancies on implementation.

Reply:
This wouldn't be cause for any issues as the data is loaded consistently with `index_col=0`. However, in a future PR I will look to change this across the repo as it might be confusing for someone viewing the data.